libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -31,6 +32,11 @@ export class BookSearchComponent {
     this.books$ = this.store.select(getAllBooks);
   }
 
+  public ngOnInit(): void {
+    // handle instant search
+    this.searchBooks();
```

Comment:
This is a fairly useless comment and I'm not sure you should be calling this function here.

Reply:
Updated.